composer.json

```diff
@@ -7,10 +7,10 @@
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.0"
+        "phpunit/phpunit": "^7.0 || ^9.0"
     },
     "require": {
-        "php": "^7.0",
+        "php": "^7.1 || ^8.0",
```

Comment:
Is it necessary to drop the support for PHP 7.0? It is usually better for the library to be compatible with as many PHP versions as possible, unless this limits the development of the library.